Experience/DistribExperienceManager.py

```diff
@@ -16,9 +14,13 @@ def push_trajectories(self, trajectories):
         if self.client is None:
             return
 
-        encoded = msgpack.packb(trajectories)
+        t1 = time.perf_counter()
+        encoded = cser.pack(trajectories)
+        print("packed {} bytes of trajectories in {:7.5f}s".format(len(encoded), time.perf_counter() - t1))
+        t2 = time.perf_counter()
 
         self.client.push_data(RedisKeys.CLIENT_EXPERIENCE_KEY, encoded, encoded=True)
+        print("pushed trajectories in {:7.5f}s".format(time.perf_counter() - t2))
```

Comment:
This code is now redundant.

I think I'll likely move the management of the `ClientTrajectoryFinalizer` (that really needs a better name) into this class, as that feels like a more appropriate home for it than the `Client` class.